pkgs/applications/misc/cobang/default.nix

```
@@ -97,8 +97,8 @@ buildPythonApplication rec {
     description = "QR code scanner desktop app for Linux";
     homepage = "https://github.com/hongquan/CoBang";
     license = licenses.gpl3Only;
-    maintainers = [ ];
+    maintainers = with maintainers; [ dvaerum aleksana ];
```

```suggestion
    maintainers = with maintainers; [ dvaerum aleksana ];
```
I'd default to alphabetical order. While the list may be small now, it could grow in the future.